spec/requests/api/v1/inventory_list_items_controller_spec.rb

```diff
@@ -0,0 +1,351 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Api::V1::InventoryListItemsController', type: :request do
+  before :all do
+    @user = create(:user)
+    @teams = create_list(:team, 2, created_by: @user)
+    create(:user_team, user: @user, team: @teams.first, role: 2)
+
+    @valid_inventory = create(:repository, name: Faker::Name.unique.name,
+                              created_by: @user, team: @teams.first)
+
+    @wrong_inventory = create(:repository, name: Faker::Name.unique.name,
+                              created_by: @user, team: @teams.second)
+    create(:repository_column, name: Faker::Name.unique.name,
+           repository: @wrong_inventory, data_type: :RepositoryTextValue)
+
+    @text_column = create(:repository_column, name: Faker::Name.unique.name,
```

Comment:
This `@text_column` is never used from what I can see. Probably, it'd be good to test against this column as well, since error needs to be returned for list item endpoints for this column.